### DISCUSSION TITLE
3006.102.1_1-gnuton0_beta1

### DISCUSSION THREAD

**Discussion by gnuton** [gnuton/asuswrt-merlin.ng#620] [Announcements]
##  GNUton's Asus Merlin changelog  ##

Date: June 29, 2024
Release: 3006.102.1_1-gnuton1

  - NEW: Added support for GT-BE98.

<hr /><em>This discussion was created from the release <a href='https://github.com/gnuton/asuswrt-merlin.ng/releases/tag/3006.102.1_1-gnuton0_beta1'>3006.102.1_1-gnuton0_beta1</a>.</em>

**Comment by valferes1970** [comment]
Good afternoon GNUton's
Compliments,

What is the difference between the two files:
GT-BE98_3006_102.1_1-gnuton0_beta1_nand_squashfs.pkgtb
71.3MB
GT-BE98_3006_102.1_1-gnuton0_beta1_rog_nand_squashfs.pkgtb
73.4MB

**Reply by gnuton** [reply]
rog image has ROG theme. The other one has just the plain blue theme.

**Reply by valferes1970** [reply]
Good morning
Compliments


Thanks for the clarification

**Reply by BailAFX** [reply]
I can't wait to try it when the router arrives.  In the meantime, thank you for your work

**Comment by valferes1970** [comment]
Good morning,
Compliments,

I have two doubts,

1st How do I block YouTube on the router for the entire network?
I've already tried Firewall - URL Filter and it doesn't work.
youtubei.googleapis.com
ytstatic.l.google.com
ytimg.l.google.com
youtube-ui.l.google.com
youtube.com
ytimg.com
googlevideo.com
youtube.googleapis.com
youtube-nocookie.com
youtu.be
m.youtube.com
On Router AX88U it works well.

2nd In Guest Network Pro, when creating a Guest network or another network, wan/lan port 1 (2.5GHz) is without internet access.

Thanks,

**Reply by valferes1970** [reply]
Sorry for my English, I'm using a translator

**Reply by valferes1970** [reply]
![Firewall - URL Filter](https://github.com/gnuton/asuswrt-merlin.ng/assets/174274054/21fbf171-f7cd-4010-84a1-2b922c4be459)

**Reply by valferes1970** [reply]
![Guest Network Pro](https://github.com/gnuton/asuswrt-merlin.ng/assets/174274054/c9a764c7-d370-4de4-a9c4-cebe1999a813)
![Guest Network Pro1](https://github.com/gnuton/asuswrt-merlin.ng/assets/174274054/96d13765-a209-453f-a670-618288d37f1d)

**Reply by valferes1970** [reply]
In Tools - System Information the temperature menu does not work
![2024-07-06_131718](https://github.com/gnuton/asuswrt-merlin.ng/assets/174274054/c6785cc4-54f6-4fe3-a734-b5d399e266fd)

**Reply by mattheweli** [reply]
Same here for temps...

**Comment by Marchcat2008** [comment]
Is there any way to fix this? The whole list has become huge...
![Снимок экрана 2024-07-08 142950](https://github.com/gnuton/asuswrt-merlin.ng/assets/175013377/d2bfb7b4-b4ff-42e5-9458-81b524d8629f)

**Comment by mattheweli** [comment]
It seems the Aura RGB won't stay off if I set it on the WebUI. If you try to change something on WLAN for example, it resets to on

EDIT: this setting (turning off) does not survive a Reboot

**Comment by mattheweli** [comment]
If I turn off the 5 GHz WiFi n.2, in the Wireless Log page 5 GHz WiFi n.2 is indicated as off, but it's on.

No 5GHz WiFi n.2 status at all. I think the page shall be edited for the difference between GT-BE98 and GT-BE98Pro

**Comment by sirHydra** [comment]
Grazie per il lavoro Antonio.

Unfortunately, this router has been a nightmare, and I'm considering returning it.

Even with the updated firmware, I'm unable to configure the 2.5G port as a WAN to establish a PPPOE connection with the ONT from my Italian fiber provider. This issue has cost me the use of one 10G port, preventing me from utilizing the router as a 10G switch. Additionally, I cannot use my old AC5300 as an AiMesh node with the BE98 as the primary router.

To make matters worse, both LAN and WiFi connections drop randomly, forcing me to reboot the router frequently. With my old AC5300, the connection was incredibly stable, and I never faced such issues in eight years.

Strangely, when I revert back to the latest official Asus firmware, the router becomes slightly more stable. I'm disappointed because I really wanted this router to work, but at this point, I'm close to giving up

**Reply by BailAFX** [reply]
Ringrazio anch'io Antonio!
This product came out with too many problems, I returned it! Sin!

**Reply by mattheweli** [reply]
> Grazie per il lavoro Antonio.
> 
> Unfortunately, this router has been a nightmare, and I'm considering returning it.
> 
> Even with the updated firmware, I'm unable to configure the 2.5G port as a WAN to establish a PPPOE connection with the ONT from my Italian fiber provider. This issue has cost me the use of one 10G port, preventing me from utilizing the router as a 10G switch. Additionally, I cannot use my old AC5300 as an AiMesh node with the BE98 as the primary router.
> 
> To make matters worse, both LAN and WiFi connections drop randomly, forcing me to reboot the router frequently. With my old AC5300, the connection was incredibly stable, and I never faced such issues in eight years.
> 
> Strangely, when I revert back to the latest official Asus firmware, the router becomes slightly more stable. I'm disappointed because I really wanted this router to work, but at this point, I'm close to giving up

Chi è Antonio?

BTW, I'm in Italy too and I'm forced to use the 10Gbit port because of the VLAN tag of my ISP.

I'm running Gnuton Beta 1 FW for 36 hours now and no drop on LAN or WiFi and I have 2 AI Mesh node (RT-86S and TUF-AX4200) and they are still working like a charm.

Do you guys like to open a thread on Fibra click forum for discussing about GT-BE98 router? I think it's quite a marvel, since it's the first one after the GT-AX6000 to not suffer about the upload issue with Z6005 ONT and 2.5 Gbit Fiber. Plus, it saturates upload also on WiFi, something I wasn't able to achieve with GT-AX6000

**Reply by Marchcat2008** [reply]
> I'm running Gnuton Beta 1 FW for 36 hours now and no drop on LAN or WiFi and I have 2 AI Mesh node (RT-86S and TUF-AX4200) and they are still working like a charm.

I confirm. The latest official firmware had WiFi failures. I installed the beta firmware - reset to factory settings and set everything up from scratch. And everything works stably.

**Reply by mattheweli** [reply]
> > Grazie per il lavoro Antonio.
> > Unfortunately, this router has been a nightmare, and I'm considering returning it.
> > Even with the updated firmware, I'm unable to configure the 2.5G port as a WAN to establish a PPPOE connection with the ONT from my Italian fiber provider. This issue has cost me the use of one 10G port, preventing me from utilizing the router as a 10G switch. Additionally, I cannot use my old AC5300 as an AiMesh node with the BE98 as the primary router.
> > To make matters worse, both LAN and WiFi connections drop randomly, forcing me to reboot the router frequently. With my old AC5300, the connection was incredibly stable, and I never faced such issues in eight years.
> > Strangely, when I revert back to the latest official Asus firmware, the router becomes slightly more stable. I'm disappointed because I really wanted this router to work, but at this point, I'm close to giving up
> 
> Chi è Antonio?
> 
> BTW, I'm in Italy too and I'm forced to use the 10Gbit port because of the VLAN tag of my ISP.
> 
> I'm running Gnuton Beta 1 FW for 36 hours now and no drop on LAN or WiFi and I have 2 AI Mesh node (RT-86S and TUF-AX4200) and they are still working like a charm.
> 
> Do you guys like to open a thread on Fibra click forum for discussing about GT-BE98 router? I think it's quite a marvel, since it's the first one after the GT-AX6000 to not suffer about the upload issue with Z6005 ONT and 2.5 Gbit Fiber. Plus, it saturates upload also on WiFi, something I wasn't able to achieve with GT-AX6000

Antonio e GNUTON! Ho visto il profilo: grazie Antonio!!!

**Comment by mattheweli** [comment]
Et voilà:

https://forum.fibra.click/d/55839-router-asus-gt-be98

**Comment by sirHydra** [comment]
After further testing, the stability issues with this firmware seem to be resolved, although I will take some more time to confirm this fully. However, the issue with the 2.5G port not functioning as WAN persists. I am eager for this to be resolved as wasting a 10G port remains the biggest problem currently

**Reply by gnuton** [reply]
@sirHydra please file bugs as single items.
you can get better updates, we can prioritize them and I won't miss anything.. and conversation if required will be less noisy.

**Comment by mattheweli** [comment]
And counting...

![Screenshot 2024-07-17 103353](https://github.com/user-attachments/assets/c2d72898-24cc-4c6a-bf2b-86cc3adc437f)

**Comment by recuerom** [comment]
Hi all BE98 Users,

Great forum to get educated and eagerly awaiting the stable version !

I have a query ; is the 2.5GB WAN reported issue related to some quirk with this Italian Fibre provider 's infrastructure or is it a general issue with the GNUTON build ?

**Reply by mattheweli** [reply]
It's not easy to answer and I suspect it's not related only to Gnuton build of the FW; on ROG forum you can find a similar discussion, but with different FW build:

https://rog-forum.asus.com/t5/gaming-network-products/rog-rapture-gt-be98-wan-lan1-not-working-as-wan/td-p/986919

**Reply by recuerom** [reply]
Thanks... the ASUS firmware has not been updated since end of April ; hopefully they will do a release soon and address this issue. I have the router ready to swap but want to wait for the stable GNUTON and if incorporates the latest Asus gpl with this fix. Seems a waste to throw 10GB when I only have 1Gb from the ISP.

**Reply by mattheweli** [reply]
I have 2.5Gbit Fiber and it seems a waste also to me using a 10Gbit port for this...

**Comment by mattheweli** [comment]
Ciao @gnuton, when you plan to release a new FW for GT-BE98?

**Reply by gnuton** [reply]
when the major bugs are fixed.

**Comment by Verwundert** [comment]
Temperature does not work and GT-BE98 get really hot with 2x 10Gbps in use. 
Any suggestion External Cooling Fan for it?

**Reply by mattheweli** [reply]
Really? I'm using 2x10Gbit (link at 2.5Gbit), but actually the CPU is "cooler" than GT-AX6000 (68 C versus 75 C)

**Reply by Verwundert** [reply]
> Really? I'm using 2x10Gbit (link at 2.5Gbit), but actually the CPU is "cooler" than GT-AX6000 (68 C versus 75 C)

I'm currently using the low fan speed so that it isn't too loud, and the CPU temperature has dropped by 11 degrees Celsius. If I use the medium or high speed, it will cool down even more. And I didn’t want to use a Coolpad for the laptop, I wanted the fans to be as hidden as possible.
![IMAGE 2024-07-28 20:37:00](https://github.com/user-attachments/assets/b9abd772-5b17-4634-802b-b4714b2092b5)
![Bildschirmfoto 2024-07-28 um 19 38 26](https://github.com/user-attachments/assets/5d32ae4b-7e38-41fc-96b4-1c45a938a83e)
![Bildschirmfoto 2024-07-28 um 20 32 12](https://github.com/user-attachments/assets/d739501d-bfe1-4938-9066-68ff1f2622c2)

**Comment by recuerom** [comment]
Has anyone tried the latest GPL for BE98  (102_36852) released 4th October 2024 ? . Looks like a major change to AIMesh. Does it resolve the 2.5Gb WAN issue (not disclosed in the documentation available).

**Reply by mattheweli** [reply]
I hope @RMerl could merge it soon on upstream and then @gnuton get it in a new release

**Reply by recuerom** [reply]
I took a punt and I upgraded one of my BE98 nodes (not the router) and it shows that link in amber as if it was 100mb , in practise I am still getting 1000mb. 

![CleanShot 2024-10-07 at 19 12 53@2x](https://github.com/user-attachments/assets/9349b591-18dd-4f73-82fb-71898375ca3e)

**Reply by RMerl** [reply]
> I hope @RMerl could merge it soon on upstream and then @gnuton get it in a new release

Not happening anytime soon, as there is no GPL to merge.  GPL updates have already missed two previous deadlines, and the latest ETA is weeks away from now.

**Reply by mattheweli** [reply]
So sad to hear that...
